perf_bench_test.go

```diff
@@ -118,83 +130,122 @@ func TestBenchRequestPerSecond(t *testing.T) {
 }
 
 // Pull a list of images
-func pullImages(t *testing.T, images []string) {
+func pullImages(t *testing.T, images []string) string {
 	for _, imageName := range images {
 		resp, _, err := funcPool.Serve(context.Background(), "plr_fnc", imageName, "record")
 		require.NoError(t, err, "Function returned error")
 		require.Equal(t, resp.Payload, "Hello, record_response!")
 	}
+
+	pidBytes, err := getFirecrackerPid()
+	require.NoError(t, err, "Cannot get firecracker pid")
+	pid := strings.TrimSpace(strings.Split(string(pidBytes), "\n")[0])
+	return pid
 }
 
 // Boot a range of VMs with given images
-func bootVMs(t *testing.T, images []string, startVMID, endVMID int) {
+func bootVMs(t *testing.T, images []string, startVMID, endVMID int, omitPid string) {
```

Comment:
omitPid is a non-intuitive name, what does it mean?

Reply:
This is the PID (flrPid above, will be renamed) of the VM instance that pulls images. We don't want it is pinned because `taskset` returns an error.